src/android_utils.py

```diff
@@ -46,7 +46,7 @@ def get_timezone_name():
 def start_service(service_name, service_args=None):
     service_args = service_args or {}
     service = autoclass(
-        "org.learningequality.Kolibri.Service{}".format(service_name.title())
+        "org.endlessos.Key.Service{}".format(service_name.title())
```

Comment:
```suggestion
    service = autoclass("org.endlessos.Key.Service{}".format(service_name.title()))
```
From the lint failure. That diff isn't quite right, but you get the idea.

Reply:
Fixed. I wasn't using the pre-commit